python-avocado.spec

```diff
@@ -1,8 +1,24 @@
 %global srcname avocado
+
+# Conditional for release vs. snapshot builds. Set to 1 for release build.
+%if ! 0%{?rel_build:1}
+%global rel_build 1
+%endif
+
+# Settings used for build from snapshots.
+%if 0%{?rel_build}
+%global gittar		%{srcname}-%{version}.tar.gz
+%else
 %if ! 0%{?commit:1}
- %define commit 1d717446d28ff852fe5de895f14c7985c63ada3e
+%global commit		817c25c2bbedb6d43e5fa2ad0a78698f1416ab49
+%endif
+%if ! 0%{?commit_date:1}
+%global commit_date	20170407
+%endif
+%global shortcommit	%(c=%{commit};echo ${c:0:7})
+%global gitrel		.%{commit_date}git%{shortcommit}
+%global gittar		%{srcname}-%{shortcommit}.tar.gz
```

Comment:
Sure, I was just asking, as I said I'm not a SPECfile expert, it just looked odd and without indentation it is really hard to follow. Anyway it seems to work well and the links look sane as well. So let's get this in and it'd be nice to suggest the `rel_build` variable name change to downstream as it's not really human friendly...